src/chaitanyaKDec19/ClassQuestions/OddEvenSumTest.java

```diff
@@ -0,0 +1,33 @@
+package chaitanyaKDec19.ClassQuestions;
+
+public class OddEvenSumTest {
+
+	public static void main(String[] args) {
+
+		if (args.length == 0) {
+			System.out.println("Please enter valid arguments i.e. numbers");
```

Comment:
Good point @ChaitanyaKDec19 
now you are thinking on various perceptive. keep it up

Reply:
Thanks a lot Harsh :) @harshvegada